src/RulesTest/Library/EdgeBrowserHasBeenDeprecatedTests.cs

```diff
@@ -29,7 +29,7 @@ public void FrameworkIssueLink_IsNotNull()
         [TestMethod]
         public void Condition_FrameworkIsNotEdge_ReturnsFalse()
         {
-            string[] nonWin32Values = { FrameworkId.DirectUI, FrameworkId.InternetExplorer, FrameworkId.WinForm, FrameworkId.WPF, FrameworkId.XAML, FrameworkId.Win32, "NotEdge" };
+            string[] nonWin32Values = { FrameworkId.DirectUI, FrameworkId.InternetExplorer, FrameworkId.WinForm, FrameworkId.WPF, FrameworkId.XAML, FrameworkId.Win32, FrameworkId.Chrome, "NotEdge" };
```

Comment:
I find 3 other test classes where we take a similar approach:
* ControlShouldSupportSetInfoWPFTests.cs
* ControlShouldSupportSetInfoXAMLTests.cs
* FrameworkDoesNotSupportUIAutomationTests.cs

Perhaps a future PR could add a utility method that would have a signature along the lines of
```
IEnumerable<string> GetFilteredFrameworks(string excludeThis, string addThis);
```
and we could use that method in all of them. Assuming that method were to use reflection to enumerate the values, adding a fields added to the `FrameworkId` class would automatically get picked up in the test classes.

Reply:
Good idea! Created a separate PR to add that utility: https://github.com/microsoft/axe-windows/pull/872